libkbfs/block_retrieval_queue.go

```diff
@@ -176,12 +176,9 @@ func (brq *blockRetrievalQueue) CacheAndPrefetch(ctx context.Context,
 	lifetime BlockCacheLifetime, hasPrefetched bool) (err error) {
 	dbc := brq.config.DiskBlockCache()
 	if dbc != nil {
-		go func() {
-			err := dbc.UpdateLRUTime(ctx, ptr.ID)
-			if err != nil {
-				brq.log.CWarningf(ctx, "Error updating metadata: %+v", err)
-			}
-		}()
+		if err := dbc.UpdateLRUTime(ctx, ptr.ID); err != nil {
```

Comment:
With this in a goroutine, the new test would flake because it'd try to call `dbc.UpdateLRUTime` after the LevelDB was closed.